tools/subdir.am

```diff
@@ -26,4 +26,5 @@ EXTRA_DIST += \
 	tools/rrlookup.pl \
 	tools/zc.pl \
 	tools/zebra.el \
+        tools/build-debian-package.sh \
```

Comment:
nit: replace spaces by tab

Reply:
Fixed. At least automake does not make tabs signficant. ;)